src/components/title/_fields/identifiers/identifiers-fields.js

```diff
@@ -13,96 +12,101 @@ import {
 
 import { FormattedMessage } from 'react-intl';
 
-export default class IdentifiersFields extends Component {
-  validateId(value) {
-    let errors;
+import { IDENTIFIERS_FIELDS_VALUE_MAX_LENGTH } from '../../../../constants';
 
-    if (!value) {
-      errors = <FormattedMessage id="ui-eholdings.validate.errors.identifiers.noBlank" />;
-    }
+const validateId = (value) => {
+  let errors;
```

Comment:
Better to rename it in `error` because in this function variable `error` get just single component as a result